core/keyboard_nav/cursor_svg.js

```diff
@@ -274,7 +274,7 @@ Blockly.CursorSvg.prototype.showWithField_ = function(curNode) {
 Blockly.CursorSvg.prototype.showWithInput_ = function(curNode) {
   var connection = /** @type {Blockly.Connection} */
       (curNode.getLocation());
-  var sourceBlock = /** @type {Blockly.BlockSvg} */ (connection.getSourceBlock());
+  var sourceBlock = /** @type {!Blockly.BlockSvg} */ (connection.getSourceBlock());
```

Comment:
is this cast needed anymore? Since getSourceBlock is now ``!Blockly.BlockSvg``

Reply:
Yes because setParent takes in a Blockly.BlockSvg not a Blockly.Block so it complains when I remove it.

Reply:
But you just changed getSourceBlock to return a ``!Blockly.BlockSvg``.

Reply:
I changed to !Blockly.Block. I was originally trying to fix this problem but when it didn't I just left the change since it seems more correct since the sourceBlock never gets set to null.

Reply:
Yep makes sense.